### PR TITLE
[docker] update bnd in docker

### PR DESCRIPTION
**Pull request by stbischof**
**Status**
- [ ] IN DEVELOPMENT
- [x] READY
- [ ] HOLD

**Description**
- update bnd in docker image  to 4.1.0, 
- use jdk to enable compile
- short description in README.md

**Checklist**
- [ ] You have added unit tests
- [x] Documentation `https://github.com/bndtools/bnd/blob/master/docker/README.md`
- [x] squash your commits in seperate `git rebase -i` and `git push -f`
- [x] Tag commit-message [exporter,wab,cli,libg,...]
- [x] Sign your work `Signed-off-by: Joe Smith <joe.smith@email.com>` or use `git commit -s`
- [ ] Related Spec:`none`
- [ ] Reference Issues: Fixes #
- [ ] Backward compatibility conflicts:
 * (no versions between (3.3.0; 4.1.0])
- [x] Known open Issues:
 * docker push to https://hub.docker.com/r/bndtoolsorg/bnd/

**Question**
@bjhargrave 
- is there a way/need to test it?

### CODE REVIEW

**Comment by bjhargrave** [comment]
> @bjhargrave
> 
>     * is there a way/need to test it?

@njbartlett You created the docker folder. Can you provide any advice here?

File: docker/build.sh [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
+VERSION=4.1.0
 # SHA-256...
-SHA=b6b68dfcd0f5ba767a202bf35eb3eb964c63e679e8217dd514dac807e6cedee8
+SHA=5ad18a5a8702fcfe42d5180ee4ba33cf4728d5f68055b2d3755a86222afac462
 
 mkdir -p tmp
 if [ ! -f tmp/bnd.jar ]; then
 	echo Downloading bnd JAR version $VERSION
-	curl https://search.maven.org/remotecontent?filepath=biz/aQute/bnd/biz.aQute.bnd/$VERSION/biz.aQute.bnd-$VERSION.jar > tmp/bnd.jar
+	curl http://central.maven.org/maven2/biz/aQute/bnd/biz.aQute.bnd/$VERSION/biz.aQute.bnd-$VERSION.jar > tmp/bnd.jar
```
Comment (bjhargrave) [inline docker/build.sh]: I don't see why you made the change from search to central? And also removed https?

File: docker/build.sh [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
 else
 	echo Already downloaded bnd JAR version $VERSION. If SHA check fails, try deleting tmp folder.
 fi
 
 echo Checking SHA...
 echo "$SHA  tmp/bnd.jar" > tmp/checksum
-shasum -a 256 -c tmp/checksum
+sha256sum -c tmp/checksum
```
Comment (bjhargrave) [inline docker/build.sh]: I would not change this either. shasum exists on more systems.

**Comment by stbischof** [comment]
i didn't got an result at the maven search site yesterday. 

i tested it manually but we can add an travis tests.

**Comment by bjhargrave** [comment]
Is this ready to merge?

**Comment by stbischof** [comment]
it is!

**Comment by bjhargrave** [comment]
Merged. Thanks.

**Pull request merged**